### PR TITLE
fix arxiv template issues

### PR DESCRIPTION
**Pull request by Athanasiamo**
This is just a small fix to enable figures from code chunk to be included without error in the arxiv templates (#332 ). 
The skeleton might also need some adapting to make it more Rmarkdown and less latex, but I did not do that, in case the format is intentionally so.

### CODE REVIEW

**Comment by cderv** [comment]
Looks good ! Thank you ! 

Could you add a new item to NEWS also ? 

Thank you !

**Comment by Athanasiamo** [comment]
should I add it under the current version number?

**Comment by cderv** [comment]
> should I add it under the current version number?

Yes 0.17 is currently the dev version

(I am just adding [magic keyword](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) so that it will close #332 when we merged this PR)

**Comment by cderv** [comment]
Thank you @Athanasiamo !

File: inst/rmarkdown/templates/arxiv/skeleton/skeleton.Rmd [hunk 1]
```diff
@@ -20,7 +20,9 @@ keywords:
   - bloo
   - these are optional and can be removed
 bibliography: references.bib
-output: rticles::arxiv_article
+output: 
+  rticles::arxiv_article:
+    keep_tex: true
```
Comment (cderv) [inline inst/rmarkdown/templates/arxiv/skeleton/skeleton.Rmd:25]: @Athanasiamo just a precision about this change I kept: 

`keep_tex = TRUE` is already the default in `arxiv_article()`. So even if you don't mention it in the YAML header, the tex file will be kept. 

FYI

**Pull request merged**